src/components/pages/donation_form/singlePageFromSections/PersonalDataSectionDonationReceipt.vue

```diff
@@ -0,0 +1,229 @@
+<template>
```

Comment:
I would call this address section rather than personal data section as we term it as address everywhere else

Reply:
we have so many different things called "section" and "address.." already, I felt the strong urge to call it something else